core/layouts/harris/layout--harris.tpl.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @file
+ * Template for the Harris layout.
+ *
+ * Variables:
+ * - $title: The page title, for use in the actual HTML content.
+ * - $messages: Status and error messages. Should be displayed prominently.
+ * - $tabs: Tabs linking to any sub-pages beneath the current page
+ *   (e.g., the view and edit tabs when displaying a node.)
+ * - $action_links: Array of actions local to the page, such as 'Add menu' on
+ *   the menu administration interface.
+ * - $classes: Array of CSS classes to be added to the layout wrapper.
+ * - $attributes: Array of additional HTML attributes to be added to the layout
+ *     wrapper. Flatten using backdrop_attributes().
+ * - $content: An array of content, each item in the array is keyed to one
+ *   region of the layout. This layout supports the following sections:
+ *   - $content['header']
+ *   - $content['top']
+ *   - $content['content']
+ *   - $content['sidebar1']
+ *   - $content['sidebar2']
```

Comment:
"sidebar1" is just "sidebar" (though this might get changed to "sidebar_first" per comments above).